src/org/omegat/util/FileUtil.java

```diff
@@ -112,15 +101,12 @@ public int compare(File f1, File f2) {
 
     /**
      * Create file backup with datetime suffix.
-     *
-     * @return Backup file
      */
-    public static File backupFile(File original) throws IOException {
+    public static void backupFile(File original) throws IOException {
```

Comment:
Should not change public method for API stability.
We should use return value in caller class.